api_core/api_request/api_request_manager.py

```diff
@@ -0,0 +1,45 @@
+# Module: api_request_manager
+import requests
+
+headers = {'Accept': 'application/json'}
+
+
+# Function: Perform GET or DELETE request
```

Comment:
This is not the way to comment a method, please use '''comment''' in all functions